data/json/items/armor/suits_clothes.json

```diff
@@ -117,10 +118,11 @@
     "symbol": "[",
     "looks_like": "jumpsuit",
     "color": "yellow",
-    "covers": [ "leg_l", "leg_r", "arm_l", "arm_r", "torso" ],
-    "coverage": 95,
-    "encumbrance": 1,
-    "max_encumbrance": 2,
+    "armor_portion_data": [
+      { "covers": [ "torso" ], "coverage": 95, "encumbrance": [ 1, 2 ] },
+      { "covers": [ "arm_l", "arm_r" ], "coverage": 50, "encumbrance": 1 },
```

Comment:
```suggestion
      { "covers": [ "arm_l", "arm_r" ], "coverage": 50, "encumbrance": [ 1, 1 ] },
```
It's not work. Pls test in game.
<img width="378" alt="3" src="https://user-images.githubusercontent.com/38557723/93617720-024f5c00-f9df-11ea-8470-4bf139a16b53.png">
`armor_portion_data` try to calc max_encumbrance itself, if you not explicitly set it. Need [1,1].
In other cases, the same thing.

Reply:
Actually that screenshot refers to `subsuit_xl`, so this line:
```
 { "covers": [ "arm_l", "arm_r" ], "coverage": 50, "encumbrance": 2 },
```